Source/ThirdParty/ANGLE/src/libANGLE/Context.cpp

```diff
@@ -10218,6 +10218,12 @@ void Context::drawPixelLocalStorageEXTDisable(const PixelLocalStoragePlane plane
     ANGLE_CONTEXT_TRY(mImplementation->drawPixelLocalStorageEXTDisable(this, planes, storeops));
 }
 
+void Context::bindMetalRasterizationRateMap(GLMTLRasterizationRateMapANGLE map)
+{
+    ANGLE_CONTEXT_TRY(mImplementation->bindMetalRasterizationRateMap(this, map));
+    mState.setVariableRasterizationRateBinding(map);
```

Comment:
I don't know ANGLE's dirty bit mechanism well, but I'd somewhat assume that the binding should be a side effect of the dirty bits being flushed. Otherwise I'm not sure how the rasterization rate map gets set/reset between context switches.